packages/gatsby-plugin-offline/README.md

```diff
@@ -143,22 +143,22 @@ const options = {
       // Use cacheFirst since these don't need to be revalidated (same RegExp
       // and same reason as above)
       urlPattern: /(\.js$|\.css$|static\/)/,
-      handler: `CacheFirst`,
+      handler: `cacheOnly`,
```

Comment:
@samlogan did you mean to change this from `cacheFirst` to `cacheOnly`?

Reply:
Good spot @m-allanson, I have opened a [new pull request](https://github.com/gatsbyjs/gatsby/pull/19926) with this update